tests/unit_tests/test_maus.py

```diff
@@ -13,7 +13,7 @@
     ValuePoolEntry,
 )
 from maus.models.message_implementation_guide import SegmentGroupHierarchy
-from unit_tests.serialization_test_helper import assert_serialization_roundtrip  # type:ignore[import]
+from tests.unit_tests.serialization_test_helper import assert_serialization_roundtrip  # type:ignore[import]
```

Comment:
```suggestion
from serialization_test_helper import assert_serialization_roundtrip  # type:ignore[import]
```

Reply:
#80

Reply:
jo können wir machen, muss ich nur die Pfade `/tests/unit_tests` und `tests/integration_tests` zum `PYTHONPATH` in der .env hinzufügen